_posts/2018-06-27-cockpit-171.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Cockpit 171
+author: pitti
+date: 2018-06-27
+tags: cockpit linux
+slug: cockpit-171
+category: release
+summary: Cockpit with VM/Kubevirt improvements, permission checks, and time localization
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 171.
+
+### Machines: Add virtual CPU configuration
+
+The Machines page can now flexibly configure the number and layout of virtual CPUs:
+
+![Machines vCPU](/images/machines-vcpu-config.png)
+
+When changing the values for an already running VM, they become effective after
+stopping and restarting it.
+
+_Thanks to Bohdan Iakymets!_
+
+### Kubernetes: Add KubeVirt pod metrics
+
+The details of a KubeVirt VM now shows the current CPU, memory, and network
```

Comment:
"The details ... shows ..." is a mismatch in number.  You either need "The details .... show ..." or (more likely) "The details page ... shows" (which is exactly the wording you use for Docker, just below).

Reply:
Thanks for spotting! Fixed.